src/main/js/main_html5.js

```diff
@@ -150,7 +150,7 @@ require("../../../html5-common/js/utils/environment.js");
     var firstPlay = true;
     var playerDimension = dimension;
     var videoDimension = {height: 0, width: 0};
-    var queuedInitialTime = 0;
+    var initialTime = { value: 0, reached: true };
```

Comment:
i vaguely remember deciding not to put it there on purpose but i dont' remember... seems right that it should be cleared.  will test it out